src/components/Article-card.jsx

```diff
@@ -0,0 +1,22 @@
+export default function ArticleCard({item}){
+    return(
+        <section className="article-card">
+            <div className="img-card">
+                <img src={item.article_img_url} alt={`Describing ${item.topic}`}/>   
+            </div>
+            <div className="content-card">
+                <p>{item.article_id}</p>
+                <p>{item.title}</p>
+                <p>{item.body}</p>
+                <p>{item.topic}</p>
+                <p>{item.created_at}</p>
+                <p>{item.votes}</p>
```

Comment:
these p tags will only display the numbers for votes and comment count - the user wont know what these numbers mean. consider changing the content inside your p tag to add a description of the number - e.g.  <p>{item.votes} votes</p> would render as 5 votes for example, rather than just 5